generators/micronaut/templates/build.gradle.ejs

```diff
@@ -230,10 +230,10 @@ dependencies {
     implementation "com.hazelcast:hazelcast"
     <%_ } _%>
     <%_ if (cacheProvider === 'hazelcast' && enableHibernateCache) { _%>
-    implementation "com.hazelcast:hazelcast-hibernate53"
+    implementation "com.hazelcast:hazelcast-hibernate53:5.2.0"
```

Comment:
```suggestion
    implementation "com.hazelcast:hazelcast-hibernate53:<%- javaDependencies['hazelcast-hibernate53'] %>"
```

And add this dependency to https://github.com/jhipster/generator-jhipster-micronaut/blob/main/generators/micronaut/resources/gradle/libs.versions.toml
So it will be updated automatically.

Reply:
Maven counterpart should be added too.
Or inject dependencies using `source.addJavaDependencies` here:
https://github.com/jhipster/generator-jhipster-micronaut/blob/38ac73bab3ab20ac8c8d5121306d4f20bd020267/generators/micronaut-cache/generator.js#L68

`source.addJavaDependencies` will add to gradle and maven.

Reply:
@mshima I worked on `source.addJavaDependencies` and implemented dependency injection for gradle taking a cue from what had already been done for the liquibase section generator.

However, I only handled the Hazelcast dependencies leaving the other cache providers as they were in the build.gradle.ejs to avoid regressions and not knowing the impacts.

What do you think?